CONTRIBUTING.md

```diff
@@ -1,49 +1,122 @@
 # OSML Contributing Guidelines
 
 Thank you for your interest in contributing to our project! This document will guide you through the process of
-contributing to our repository using a release-based GitFlow workflow. Please read the guidelines carefully in order to
+contributing to our repository using a release-based GitFlow workflow. Please read the guidelines carefully to
```

Comment:
... using a trunk based development workflow.

In multiple places be careful of the terms used. `Trunk Based Development` and `GitFlow` are well established branching strategies. Our team has decided to move from GitFlow to TBD. I have no idea what `Trunk-Based GitFlow` is.

Reply:
Will update